content/collections/workflow/en/experiment-learnings.md

```diff
@@ -13,59 +13,113 @@ You’ve designed your experiment, rolled it out to your users, and given them e
-    {{/partial:admonition}}
-
-3. If desired, adjust the experiment’s **confidence level**. The default is 95%. You can also [choose between a sequential test and a T-test](/docs/feature-experiment/workflow/finalize-statistical-preferences).   
+To learn more about exposure events, see [Event Tracking](/docs/feature-experiment/under-the-hood/event-tracking).  
+#}} 
+    
+Click _Chart Controls_ to see the chart definition. 
+    
+Click *Open in Chart* to open a copy of the Experiment Results in a new chart.
+    
+{{partial:admonition type='note'}}
+If you are running an A/B/n test, Amplitude Experiment displays the confidence interval / p-value for the control against each treatment individually. To instead see the comparison between two non-control treatments, either change the control variant, or open the test in Analytics and create a chart using the two treatments you're interested in.
+{{/partial:admonition}}
+
+If desired, adjust the experiment’s **confidence level**. The default is 95%. You can also [choose between a sequential test and a T-test](/docs/feature-experiment/workflow/finalize-statistical-preferences).   
   
-    {{partial:admonition type='note'}}
-    Lowering your experiment’s confidence level will make it more likely that your experiment achieves statistical significance, but the trade-off is that doing so increases the likelihood of a false positive.
-    {{/partial:admonition}}
+{{partial:admonition type='note'}}
+Lowering your experiment’s confidence level makes it more likely that your experiment achieves statistical significance, but the trade-off is that doing so increases the likelihood of a false positive.
+{{/partial:admonition}}
 
-4. Set the **time frame** for your experiment analysis, either from the selection of pre-set durations, or by opening the date picker and choosing a custom date range.
+## Diagnostics card
 
-    The tables, graphs, and charts shown in the Analysis section are explained in depth in the articles on [understanding the Experiment Analysis view](/docs/feature-experiment/analysis-view) and [interpreting the cumulative exposures graph in Amplitude Experiment](/docs/feature-experiment/advanced-techniques/cumulative-exposure-change-slope).
+The Diagnostics card provides information about how your experiment is delivering. It shows charts about:
 
-    {{partial:admonition type='note'}}
-    Amplitude Experiment needs something to compare your control to in order to generate results. If you neglect to include **both** the control and **at least one** variant, your chart will not display anything.
-    {{/partial:admonition}}
+* Assignment events (cumulative and non-cumulative)
+* Exposure events (cumulative and non-cumulative)
+* Assignment to exposure conversion
+* Variant jumping
+* Anonymous exposures
```

Comment:
We probably need to explain how to read these charts but that can happen in another PR